include/ConfigManager.h

```diff
@@ -137,15 +137,15 @@ class LMMS_EXPORT ConfigManager : public QObject
 		return m_ladspaDir;
 	}
 
-	const QString & sf2Dir() const
+	const QString & soundFontDir() const
```

Comment:
(Non-blocking) Per the [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers):

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

```suggestion
	const QString& soundFontDir() const
```

Might as well update this if you're already touching this line. Also, the whole function can go on a [single line](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces) since it is just a `return thing`.